CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Line selection will now expand across wrapped lines
 - The default value for `draw_bold_text_with_bright_colors` is now `false`
 - Mirror OSC query terminators instead of always using BEL
+- Beam and Underline cursors width computation to be more visible
```

Comment:
This affects the box cursor too.

```suggestion
- Increased Beam, Underline and Box cursors widths
```

Reply:
Ehm, “Hollow block”, nit Block, we’re not passing line_width to a block cursor, I’ve decided to avoid mentioning Block cursor tbh.

Reply:
Why? This affects the hollow block cursor, so we should mention it.

Reply:
Though I'd probably recommend saying `line widths` instead of just `widths`. Because width is misleading for underline and hollow block.